frontend/cypress/support/commands.js

```diff
@@ -27,22 +27,22 @@ Cypress.Commands.add('interceptAllRequests', () => {
     disableRequestCache,
   ).as('menuEntriesRequest');
 
-  cy.intercept(
-    {
-      url: '/static-journeys/journeysPageIndex.json',
-      method: 'GET',
-      middleware: true,
-    },
-    disableRequestCache,
-  ).as('journeyListRequest');
-  cy.intercept({ method: 'GET', url: '/static-journeys/1.json', middleware: true }, (req) => {
+  // TODO: remove comments when API is ready
+  // cy.intercept(
+  //   {
+  //     url: '/static-journeys/journeysPageIndex.json',
+  //     method: 'GET',
+  //     middleware: true,
+  //   },
+  //   disableRequestCache,
+  // ).as('journeyListRequest');
```

Comment:
I may wrong, but shouldn't it be uncommented? Because it should be available when `NEXT_PUBLIC_STATIC_JOURNEYS` is `true`, right?

Reply:
We will have to have a env variable for the tests then. Do you think they are going to fail in production?